### PR TITLE
My Quiz Entry 

### PR DESCRIPTION
**Pull request by garrett-vangilder**
This is my quiz submission.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    </section>
+    <section class="pricing">
+      <h2>Pricing</h2>
+        <p>Today's Price: $200</p>
+        <p>5 Easy Payments: $50</p>
+        <p>Buy one tent get the second tent 50% off!</p>
+     </section>
+  </article>
+</body>
+</html>
```
Comment (sscotth) [inline index.html]: Always end a file with a new line character. We will discuss this in class.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+</head>
+
+
+<body>
+  <article class="product">
+    <header>
+      <h1>Ultralight 2-Person Tent</h1>
+    </header>
+    <section>
+      <img class="resize" src="img/2-person-tent.jpeg">
```
Comment (sscotth) [inline index.html]: All `img` tags need an `alt` attribute. Also can you use a more semantic class name like "product-image"

File: quiz.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+
+.product {
+  border: 1px solid lightblue;
+  border-radius: 5px;
+  margin:.75em;
+  padding: 10px;
+  font-family: 'Lora', serif;
+  clear:
```
Comment (sscotth) [inline quiz.css]: This is not valid.

File: quiz.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+ .product footer: {
+  font-size: 50%;
+}
+
+@media (min-width: 600px) {
+  body {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (sscotth) [inline quiz.css]: Consider using floats instead of flexbox for wrapping.

**Comment by sscotth** [comment]
You are missing the images in the pull request make sure they are committed. Also make sure that your cards are 3 across at moderate resolutions.  When I drop below 1396 px wide, I only see two across.

File: quiz.css (hunk 4 above)
Comment (garrett-vangilder) [inline quiz.css]: Hey Scott, I'm having problems using a float. I changed it all to floats, but then it created a really strange margin on the left side of the third card. All other changes are fine, and have been implemented.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+<html>
+<head>
+  <title>Static Web Quiz</title>
+  <link href="https://fonts.googleapis.com/css?family=Lora:400,700|Open+Sans:400,700" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head> 
+
+
+<body>
+  <article class="product left">
```
Comment (sscotth) [inline index.html:11]: what does the left, right, and center class do?

File: quiz.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+/*.left {
+  float:left;
+}
+.right {
+  float: right;
+}
+
+.product {
+    width: 30%;
+    margin: 10em; }*/
```
Comment (sscotth) [inline quiz.css]: Did you mean to leave this comment in the PR?

File: quiz.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
+body {
+  margin-left: 1rem;
+}
+body .product {
```
Comment (sscotth) [inline quiz.css]: why `body .product` and not simply `.product`?

File: quiz.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  margin: 5px;
+  display: inline-block;
```
Comment (sscotth) [inline quiz.css]: You are using inline-block here, but using flex down below. You cannot do both.

File: quiz.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.product {
+  border: 1px solid lightblue;
+  border-radius: 5px;
+  margin:.75em;
+  padding: 10px;
+  font-family: 'Lora', serif;
+  margin-top: 5em;
```
Comment (sscotth) [inline quiz.css:13]: You have product margins in 3 different places. Pick one

File: quiz.css [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  padding-bottom: 15px;
+}
+
+.product-image{
+    width:100%; 
+    display: block;
+    height: auto;
+    margin: auto;
```
Comment (sscotth) [inline quiz.css]: is the height and margin doing anything?

File: quiz.css [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+  font-family: 'Lora', serif;
+  font-weight: 700;
+}
+
+.availability {
+  background: grey;
+  color:yellow;
+}
+
+ .product footer: {
```
Comment (sscotth) [inline quiz.css]: You have a misplaced colon. Also, you are being overly specific, simply use `footer`

**Comment by sscotth** [comment]
Other than some minor things I just added, you have met the minimum requirements. However, I mentioned before about the 2 vs 3 across problem, now the problem exists at 600-673px. If you want to fix it, either make sure your breakpoints are at the right places to jump from 3 to 1 across or simply center the content on the page so that when it goes to 2 across it looks as if it is intended.

**Pull request merged**